### PR TITLE
[Doc] add kafka connector release

### PR DESCRIPTION
**Pull request by hellolilyliuyi**
Fixes #issue

## What type of PR is this:

- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [x] Doc
- [ ] Tool

Does this PR entail a change in behavior?

- [ ] Yes, this PR will result in a change in behavior.
- [x] No, this PR will not result in a change in behavior.

If yes, please specify the type of change:

- [ ] Interface/UI changes: syntax, type conversion, expression evaluation, display information
- [ ] Parameter changes: default values, similar parameters but with different default values
- [ ] Policy changes: use new policy to replace old one, functionality automatically enabled
- [ ] Feature removed
- [ ] Miscellaneous: upgrade & downgrade compatibility, etc.

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:

- [x] I have checked the version labels which the pr will be auto-backported to the target branch
  - [x] 3.2
  - [x] 3.1
  - [x] 3.0
  - [x] 2.5

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [BE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [FE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [FE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [BE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [FE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [BE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [FE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [BE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [FE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [BE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [FE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [BE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [FE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [BE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [FE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [BE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

File: docs/ecosystem_release/kafka_connector.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Kafka connector
```
Comment (amber-create) [inline docs/ecosystem_release/kafka_connector.md:1]: I think the title should be "Kafka connector releases"

**Review by evelynzhaojie — APPROVED** [review]

File: docs/ecosystem_release/kafka_connector.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+## Notifications
+
+**User guide:** [Load data using Kafka connector](../loading/Kafka-connector-starrocks.md)
+
+**Naming format of the compressed file:** `starrocks-kafka-connector-${connector_version}.tar.gz`
```
Comment (amber-create) [inline docs/ecosystem_release/kafka_connector.md:7]: -> Compressed file naming convention

File: docs/ecosystem_release/kafka_connector.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+**Download link of the compressed file:** [starrocks-kafka-connector-1.0.0.tar.gz](https://releases.starrocks.io/starrocks/starrocks-kafka-connector-1.0.0.tar.gz)
```
Comment (amber-create) [inline docs/ecosystem_release/kafka_connector.md:9]: Compressed file download link

File: docs/ecosystem_release/kafka_connector.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+**Version requirements:**
+
+| Kafka Connector | StarRocks | Java |
+| --------------- | --------- | ---- |
+| 1.0             | 2.1 and later | 8    |
+
+## Release note
```
Comment (amber-create) [inline docs/ecosystem_release/kafka_connector.md:17]: suggest using plural form "notes"

File: docs/ecosystem_release/kafka_connector.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+
+### 1.0
+
+**Release date:** June 25, 2023
+
+**Features**
```
Comment (amber-create) [inline docs/ecosystem_release/kafka_connector.md:23]: -> New features

**Review by amber-create — APPROVED** [review]

**Comment by hellolilyliuyi** [comment]
@mergify backport  branch-3.2 branch-3.1 branch-3.0 branch-2.5

**Comment by hellolilyliuyi** [comment]
@mergify backport  branch-3.2

**Pull request merged**